Elevel/Application/Features/TopicCommands/CreateTopicCommand.cs

```diff
@@ -37,6 +41,15 @@ public Handler(IApplicationDbContext context, IMapper mapper)
             public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
             {
                 var topic = _mapper.Map<Topic>(request);
+                var validator = new Validator();
+
+                validator.Validate(topic, options =>
```

Comment:
validator doesn't need here, it will be executed automatically by mediatr pipeline